jenkins/open-mpi-build-script.sh

```diff
@@ -166,18 +166,30 @@ echo "--> Building commit ${sha1}"
 if test -f autogen.pl; then
     echo "--> running ./autogen.pl ${AUTOGEN_ARGS}"
     ./autogen.pl ${AUTOGEN_ARGS}
+    if test $? -ne 0; then
```

Comment:
set -e is called at the top of this script; this change won't do anything.

Reply:
i see, will fix from now

Reply:
done !